lib/stat_tracker.rb

```diff
@@ -29,9 +29,13 @@ def total_games
     @games.count { |game| game }.to_f
   end
 
-  # def percentage_home_wins
+  def percentage_home_wins
+    home_wins = @game_teams.count do |game|
+      game.result == "WIN" && game.hoa == "home"
+    end
 
-  # end
+    (home_wins / self.total_games).round(2)*100
```

Comment:
liking the use of self here, I think we can utilize this a bit more in the upcoming methods.